Kudu.SiteManagement/SiteManager.cs

```diff
@@ -171,7 +171,7 @@ private static List<string> GetSiteUrls(IIS.Site site)
                 {
                     try
                     {
-                        DeleteSiteAsync(applicationName).Wait();
+                        await DeleteSiteAsync(applicationName);
```

Comment:
It's ok, then. I'll keep that commit separate, and squash all the others related to this PR.